packages/flutter_tools/test/widget_preview_scaffold.shard/widget_preview_scaffold/lib/src/widget_preview.dart

```diff
@@ -31,15 +31,33 @@ class WidgetPreview {
   const WidgetPreview({
     required this.builder,
     required this.scriptUri,
+    required this.line,
+    required this.column,
     required this.previewData,
     required this.packageName,
   });
 
+  @visibleForTesting
```

Comment:
The two copies of these files confuse me - is there any way the test could cause this to be generated so a) they can't be out of sync and b) simplify reviews?